boards/xtensa/intel_adsp_ace15_mtpm/doc/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _intel_adsp_ace15:
+
+Intel ADSP ACE 1.5
+##################
+
+Overview
+********
+
+This board configuration is used to run Zephyr for ADSP applications on Intel
+Meteor Lake microprocessors. It is most commonly used to run version 1.5 of
+Intel's ACE firmware. Refer to :ref:`intel_adsp_generic` for more details.
```

Comment:
s/firmware/hardware/ Version 1.5 is version of the hardware. Maybe reword as: "This board configuration is used to run Zephyr on the Intel ACE 1.5 Audio DSP. This configuration is present for example on Intel Meteor Lake microprocessors."